core-services/document-grounding/src/main/java/com/sap/ai/sdk/grounding/client/VectorApi.java

```diff
@@ -914,11 +914,11 @@ public DocumentResponse getDocumentById(
    *
    * @param aiResourceGroup Resource Group ID
    * @param textSearchRequest The value for the parameter textSearchRequest
-   * @return SearchResults
+   * @return VectorSearchResults
    * @throws OpenApiRequestException if an error occurs while attempting to invoke the API
    */
   @Nonnull
-  public SearchResults search(
+  public VectorSearchResults search(
```

Comment:
(Comment)

Also here we may need a BLI for adding an integration test for vector searching. Low priority.